dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GithubAppCheck.java

```diff
@@ -122,38 +133,20 @@ protected boolean isGithubAppEnabledOnRepositoryWithGitApi(String fullRepoName)
     protected boolean isGithubAppEnabledOnRepositoryWithRenovateApi(String fullRepoName) {
         try {
             String apiEndpoint = appServerApiEndpoint + "/api/repos/" + fullRepoName;
-            URL url = new URL(apiEndpoint);
-            HttpURLConnection conn = (HttpURLConnection) url.openConnection();
-            conn.setRequestMethod("GET");
-            conn.setRequestProperty("Accept", "application/json");
-            conn.setRequestProperty("authorization", "Bearer " + appServerApiToken);
-            conn.connect();
-            
-            Integer responseCode = conn.getResponseCode();
-            if (responseCode != 200) {
-                throw new RuntimeException("HttpResponseCode: " + responseCode);
-            } else {
-                String inline = "";
-                Scanner scanner = new Scanner(url.openStream());
-                while (scanner.hasNext()) {
-                    inline += scanner.nextLine();
-                }
-                scanner.close();
-
-                //Using the JSON simple library parse the string into a json object
-                JSONParser parse = new JSONParser();
-                JSONObject dataObject = (JSONObject) parse.parse(inline);
-                String appInstallationState = (String) dataObject.get("state");
-                String appActivationStatus = (String) dataObject.get("status");
-
-                // Return app installation and activation status
-                if (appInstallationState == "installed" && appActivationStatus == "activated") {
-                    return true;
-                }
-                return false;
+            HttpGet httpGet = new HttpGet(apiEndpoint);
+            httpGet.setHeader("Authorization", appServerApiToken);
+            httpGet.setHeader("Accept", "application/json");
+            CloseableHttpResponse response = httpClient.execute(httpGet);
```

Comment:
Use try-with-resources to eliminated the finally block, https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
@cjneasbi The doc you shared yesterday recommends the `finally` block to close the response object, see more: https://www.tutorialspoint.com/apache_httpclient/apache_httpclient_closing_connection.htm

Reply:
@cjneasbi This has been resolved